model/trace/database.yaml

```diff
@@ -240,11 +243,20 @@ groups:
             sensitive information.
         brief: The request body for a [search-type query](https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html), as a json string.
         examples: [ '"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"' ]
+      - ref: db.collection.name
+        requirement_level: recommended
+        brief: The index or data stream against which the query is executed.
+        note: >
+          If the query targets multiple indices or data streams, then the name of those should be added as a comma separated list.
```

Comment:
Any thoughts on this?

Other parts of the db spec typically use the 1. item and drop the rest. Same for [`db.collection.name` in the general db span spec](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md#common-attributes).

I don't see a huge benefit of comma separating index names here - that's already captured in `db.elasticsearch.path_parts.<key>`. I think it's better to be consistent with the general spec here, so I'd change this to only capture the 1. index name.

Reply:
> I think it's better to be consistent with the general spec here

👍